receiver/opencensusreceiver/config.go

```diff
@@ -33,13 +33,13 @@ type Config struct {
 	CorsOrigins []string `mapstructure:"cors_allowed_origins"`
 }
 
-func (rOpts *Config) buildOptions() ([]ocOption, error) {
+func (cfg *Config) buildOptions() ([]ocOption, error) {
 	var opts []ocOption
-	if len(rOpts.CorsOrigins) > 0 {
-		opts = append(opts, withCorsOrigins(rOpts.CorsOrigins))
+	if len(cfg.CorsOrigins) > 0 {
+		opts = append(opts, withCorsOrigins(cfg.CorsOrigins))
 	}
 
-	grpcServerOptions, err := rOpts.GRPCServerSettings.ToServerOption()
+	grpcServerOptions, err := cfg.GRPCServerSettings.ToServerOption()
```

Comment:
Not sure this change belongs to this PR

Reply:
golint requires the same receiver name. I changed it to `cfg`